openedx/core/djangoapps/ace_common/template_context.py

```diff
@@ -38,4 +38,5 @@ def get_base_template_context(site):
         'social_media_urls': get_config_value_from_site_or_settings('SOCIAL_MEDIA_FOOTER_URLS', site=site),
         'mobile_store_urls': get_config_value_from_site_or_settings('MOBILE_STORE_URLS', site=site),
         'logo_url': get_logo_url_for_email(),
+        'site_configuration_values': site.configuration.site_values if hasattr(site, 'configuration') else {},
```

Comment:
I really don't like this as a single line if/else, but I'm not sure the best way to make this a clearer statement.

Reply:
I've removed inline if/else and initialized a variable beforehand by specifying an explicit condition.